README.md

```diff
@@ -3,25 +3,49 @@ Azure Functions prepared for the Mapping Viz Tools project. This repo contains [
\ No newline at end of file
+Although you could use [Azure Slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots) to handle different environments, you can also deploy the same function at a different location (eg. `mapping-viz-functions-staging` vs `mapping-viz-functions`) and set a different connection string at each endpoint to handle staging and production databases.
+
+## Database Maintenance
+The current schema is saved in db/schema.sql. This files are generated using [pg_dump](https://www.postgresql.org/docs/9.5/app-pgdump.html) (version 9.5.23). There are also files containing test data for local development, and the full dataset for back-up purposes - those live in db/data. The full_data.sql file is compressed using the `pg_dump` command's built in compression (using the flag `--compress=9`).
+
+We have currently decided to go with a bare-bones approach to database maintenance and schema migrations, due to the fact that we do not foresee the schema changing much going forward. If the trajectory of the project changes, it may be helpful to make use of a package or library to manage the database.
+
+### Schema Migrations
+The current process for updating the schema is as follows:
+1. Add a .sql file named 'migration10-13-20.sql' (replace with current date) to the db/schema_migrations folder. This file should contain the SQL commands that need to be executed in order to make the schema change, e.g. `ALTER TABLE`, etc.
+2. Put up a PR to have the schema changes reviewed. Once reviewed and merged, apply the changes to the staging and production databases. You can do so by connecting to the remote database using `psql`, then in the `psql` prompt, enter the command `\include \path\to\file\migration10-13-20.sql` (see more on how to use the `\include` command in the [psql documentation](https://www.postgresql.org/docs/9.5/app-psql.html)).
+3. Once the schema changes have been applied, perform a `pg_dump` of the schema for both environments, using the `--no-owner` and `--no-acl` flags. To generate a file with just the schema, use the -s flag like so: `pg_dump -s mapping_viz_production > schema.sql`. Replace db/schema.sql with the dump file in the repo. Also perform a dump of the data if necessary (using the data-only flag `-a`) and replace the test_data.sql and full_data.sql files in db/data. Note that the test_data.sql file only contains 100 records for each table and is not compressed, while the full_data.sql file contains all of the data and is compressed. To avoid version conflict with the remote postgres database, it's best to use pg_dump version 9.5. 
```

Comment:
add the no acl / owner flags to your example cmd
`pg_dump -s -h db_host -d mapping_viz_production -U username -W password --no-owner --no-acl  > ./db/schema.sql`

Reply:
I'd change the following to specify the version should match the postgres server major / minor versions, e.g. if postgres is 9.5.24 then use pg_dump 9.5 (ignore patch)
> To avoid version conflict with the remote postgres database, it's best to use pg_dump version 9.5.